indigo_app/views/works.py

```diff
@@ -431,9 +431,9 @@ def form_valid(self, form):
 
         # do normal things to amend work
         self.object.updated_by_user = self.request.user
+        result = super().form_valid(form)
```

Comment:
This isn't strictly necessary, but a good pattern is to save the form before using related attributes on the updated object.